src/instructlab/model/train.py

```diff
@@ -520,20 +580,340 @@ def get_files(directory: str, pattern: str) -> list[str]:
         # pull the trainrandom.randinting and torch args from the flags
         # the flags are populated from the config as a base.
         params = ctx.params
+        ds_args = DeepSpeedOptions(**params)
+        lora_args = LoraOptions(**params)
+        train_args = TrainingArgs(**params)
+        torch_args = TorchrunArgs(**params)
+        train_args.deepspeed_options = ds_args
+        train_args.lora = lora_args
```

Comment:
We should still be using the `map_train_to_library` function for this:


```suggestion
        train_args, torch_args = map_train_to_library(params)
```

Reply:
ack